tests/TestCase/View/Helper/FormHelperTest.php

```diff
@@ -4481,13 +4481,13 @@ public function testRadio(): void
 
         $result = $this->Form->radio('Model.field', ['option A', 'option B'], ['name' => 'Model[custom]']);
         $expected = [
-            ['input' => ['type' => 'hidden', 'name' => 'Model[custom]', 'value' => '']],
-            ['label' => ['for' => 'model-custom-0']],
```

Comment:
We can't change these id values. While they are inconsistent with other fields user-land code will be relying on these attribute values and us changing them will constitute a breaking change.